File: Programming Basics/JavaScript/2. Simple Calculations/06. Circle Area and Perimeter.js
function Solve([arg]){
    let r = Number(arg);
    let area = Math.PI * r * r;
    let perimeter = 2 * Math.PI * r;
    console.log(`Area = ${area}`);
    console.log(`Perimeter = ${perimeter}`); 
}

Solve([3]);

File: Programming Basics/02. Simple Calculations/06. Circle Area and Perimeter/06. Circle Area and Perimeter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _6.Circle_Area_and_Perimeter
{
    class Program
    {
        static void Main(string[] args)
        {
            var r = double.Parse(Console.ReadLine());
            double area = r * r * Math.PI;
            double perimeter = 2 * r * Math.PI;
            Console.WriteLine($"Area = {area}");
            Console.WriteLine($"Perimeter = {perimeter}");
        }
    }
}
